catalog/app/containers/Bucket/PackageDialog/PackageCreationForm.tsx

```diff
@@ -235,6 +301,18 @@ function PackageCreationForm({
       })),
     )
 
+    if (!entries.length) {
```

Comment:
I think, it makes more sense to check if package has README rather than if package has no files

Reply:
addressed in another comment